WalletWasabi.Fluent/ViewModels/Wallets/Settings/WalletCoinJoinSettingsViewModel.cs

```diff
@@ -50,6 +52,13 @@ public WalletCoinJoinSettingsViewModel(UiContext uiContext, IWalletModel walletM
 		_wallet = walletModel;
 		_autoCoinJoin = _wallet.Settings.AutoCoinjoin;
 		_plebStopThreshold = _wallet.Settings.PlebStopThreshold.ToString();
+
+		_excludeCoinFromCoinjoinThreshold =
+			_wallet.Settings.ExcludeCoinFromCoinjoinThreshold.ToUnit(MoneyUnit.BTC) ==
+			Constants.DefaultExcludeCoinFromCoinjoinThreshold
+				? ""
```

Comment:
I don't want to display 21000000 BTC by default because it feels ugly, I prefer leaving blank. This is the cost of not using `null`